Makefile

```diff
@@ -214,9 +214,10 @@ release:
 ifneq (,$(QUALIFIED_TAG))
 	docker pull $(DATAMGR_IMAGE):$(QUALIFIED_TAG)
 	docker pull $(PLUGIN_IMAGE):$(QUALIFIED_TAG)
-	@docker tag $(DATAMGR_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(BACKUPDRIVER_BIN):$(RELEASE_TAG)
-	@docker tag $(DATAMGR_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(DATAMGR_BIN):$(RELEASE_TAG)
-	@docker tag $(PLUGIN_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(PLUGIN_BIN):$(RELEASE_TAG)
+	docker pull $(BACKUPDRIVER_IMAGE):$(QUALIFIED_TAG)
+	docker tag $(BACKUPDRIVER_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(BACKUPDRIVER_BIN):$(RELEASE_TAG)
+	docker tag $(DATAMGR_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(DATAMGR_BIN):$(RELEASE_TAG)
+	docker tag $(PLUGIN_IMAGE):$(QUALIFIED_TAG) $(RELEASE_REGISTRY)/$(PLUGIN_BIN):$(RELEASE_TAG)
```

Comment:
This is Lintong's PR that is merged already: https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/229/files

Looks like a rebase error.